src/skin/options.html

```diff
@@ -205,7 +205,7 @@ <h4 id="privacy-settings-header" class="i18n_options_privacy_settings" style="di
           <input type="checkbox" id="disable-google-nav-error-service-checkbox">
           <span>
             <span class="i18n_options_disable_google_nav_error_service"></span>
-            <a href="https://www.google.com/chrome/privacy/#how-chrome-handles-your-information" target="_blank"><span class="ui-icon ui-icon-circle-b-help"></span></a>
+            <a href="https://www.google.com/chrome/privacy/archive/20200520/" target="_blank"><span class="ui-icon ui-icon-circle-b-help"></span></a>
```

Comment:
Let's use the most recently available archive, and let's deep link to the relevant section:

https://www.google.com/chrome/privacy/archive/20220623/#how-chrome-handles-your-information